### DISCUSSION TITLE
multiple labels for one ScaleSet

### DISCUSSION THREAD

**Discussion by David9902** [actions/actions-runner-controller#3621] [Ideas]
Hi, 
one more feature request from my side. 
It would be great to have the possibility for multiple labels for one ScaleSet. 
Allowing to have different "runs-on" for the same ScaleSet in the background. 

E.g having linux, ubuntu, ubuntu-22.04 for a ScaleSet running Ubuntu-22.04

Thanks in advance.

**Comment by julian-waibel** [comment]
This is already requested here: https://github.com/actions/actions-runner-controller/discussions/3340
